tests/kernel/smp/src/main.c

```diff
@@ -1004,8 +1014,12 @@ static void inc_global_cnt(void *a, void *b, void *c)
 	}
 }
 
-static int run_concurrency(int type, void *func)
+static int run_concurrency(void *p1, void *p2, void *p3)
 {
+	ARG_UNUSED(p3);
+
+	int type = p1;
```

Comment:
isn't this causing a warning ? you probably should use `POINTER_TO_INT(p1)`

Reply:
done